app/classifier/tasks/metadata-based-feedback.cjsx

```diff
@@ -0,0 +1,69 @@
+React = require 'react'
+
+module.exports = React.createClass
+  displayName: 'MetadataBasedFeedback'
+
+  withinTolerance: (userX, userY, metaX, metaY, tolerance) ->
+    distance = Math.sqrt((userY - metaY)**2 + (userX - metaX)**2)
+    isWithinTolerance = distance < tolerance
+    return isWithinTolerance
+
+  withinAnyTolerance: (userX, userY) ->
+    for key, value of @props.subject?.metadata
+      if key.indexOf(@props.metaSimCoordXPattern) == 0
```

Comment:
Generally in javascript, `===` should be used. In coffeescript, `is` can also be used. [Here's a good summary about why](http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons)